eth2near/contract_wrapper/src/eth_client_contract.rs

```diff
@@ -111,19 +107,6 @@ impl EthClientContract {
 }
 
 impl EthClientContractTrait for EthClientContract {
-    fn get_last_submitted_slot(&self) -> u64 {
-        return self.last_slot;
-    }
-
-    fn is_known_block(&self, execution_block_hash: &H256) -> Result<bool, Box<dyn Error>> {
```

Comment:
I've returned the `is_known_execution_header` method in the last commits, so you can keep it if you need it.

Reply:
I don't need it in the current implementation. I  think It is ok don't return this function.